ix-dev/stable/ix-app/migrations/migration_helpers/kubernetes_secrets.py

```diff
@@ -0,0 +1,15 @@
+def get_value_from_secret(secrets={}, secret_name="", key=""):
```

Comment:
kwargs should be initialized to `None` (it's the paradigm we follow in middleware repo).

Inside your function it'd look like this:
```
def get_value_from_secret(secrets=None, secret_name=None, key=None):
    secrets = secrets if secrets else dict()
    secret_name = secret_name if secret_name or ""
    key = key if key else ""